docs/instrumenting.asciidoc

```diff
@@ -383,6 +384,29 @@ func main() {
 }
 ----
 
+[[builtin-modules-apmgopgv9]]
+==== module/apmgopgv9
+Package apmgopgv9 provides a means of instrumenting http://github.com/go-pg/pg[go-pg] database operations for v9 and above.
```

Comment:
"v9 and above" might be misinterpreted to mean it'll support v10, whenever that comes along. Can you please drop the "and above"?

Reply:
Fixed now.